tests/safety_replay/test_safety_replay.py

```diff
@@ -23,6 +23,8 @@
   ("5b7c365c50084530_2020-04-15--16-13-24--3--rlog.bz2", Panda.SAFETY_HYUNDAI, 0),  # HYUNDAI.SONATA
   ("610ebb9faaad6b43|2020-06-13--15-28-36.bz2", Panda.SAFETY_HYUNDAI_LEGACY, 0),  # HYUNDAI.IONIQ_EV_LTD
   ("5ab784f361e19b78_2020-06-08--16-30-41.bz2", Panda.SAFETY_SUBARU_LEGACY, 0),  # SUBARU.OUTBACK
+  ("95441c38ae8c130e_2020-06-08--12-10-17.bz2", Panda.SAFETY_SUBARU_LEGACY, 1),  # SUBARU.FORESTER
```

Comment:
`SUBARU.FORESTER` is the global model

Reply:
changed the comments to match preglobal PR labels